vixenNetwork.yml

```
@@ -0,0 +1,25 @@
+---
+name: "Vixen/Blacked/Tushy"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - "https://www.blackedraw.com/"
+      - "https://www.blacked.com/"
+      - "https://www.tushy.com/"
+      - "https://www.tushyraw.com/"
+      - "https://www.vixen.com/"
```

```suggestion
      - "https://www.vixen.com/"
      - "https://www.deeper.com/"
```

Deeper can be added to this list.